ceno_zkvm/src/e2e.rs

```diff
@@ -1166,8 +1314,7 @@ pub fn generate_witness<'a, E: ExtensionField>(
                     .assign_init_table_circuit(
```

Comment:
In fact we can just avoid this step as we only assign static init tables in the 0th shard.

Reply:
although literally we can omit this in shard id > 0 but in some place we need all chip presented with `empty` RMM so we still need it, e.g. remove this cause `mock_prover` failed